.github/copilot-instructions.md

```
@@ -0,0 +1,70 @@
+# Copilot instructions for this repository
+
+This repo is a lightweight JavaScript (ESM) API testing project for https://reqres.in built on Mocha + Chai + Axios with strong quality gates. Use these rules to be productive and respect local conventions.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)